AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -48,6 +48,8 @@ gds.it###oggiinedicola
 namu.wiki##div[class="c"][data-v-37880e75]
 board.namu.wiki,arca.live###newsRank
 arca.live###namuRank
+! https://github.com/AdguardTeam/AdguardFilters/issues/79295
+toroox.com#?#.box-main > div.row > div.col-md-6:has(> div.link-details > div.panel)
```

Comment:
Is not it covered by `toroox.com,payskip.org#$#.link-details > div.panel { display: none !important; }`?
Btw why css is requred here?

Reply:
This rule is necessary to remove the leftover

Reply:
Yeah